util/dvsim/LintParser.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Helper class for parsing lint reports into a generic hjson format.
```

Comment:
We can do that as a follow up yes.
This lint parser update in this PR actually helps with that since it aligns all three lint flows and CDC to use a common class and format for the results.